etc/notebooks/examples/prep/5_utils.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
This should be under `etc/notebooks/demos/...

And maybe name the main demo ipynb something more descriptive